cmake/InstallLayout.cmake

```diff
@@ -246,14 +246,21 @@ else()
   set(prerelease_string "-1")
 endif()
 
-
+#############
+# Filenames #
+#############
+set(unversioned_postfix "${PROJECT_VERSION}${prerelease_string}")
 # RPM filenames
-set(rpm-clients-filename "foundationdb-clients-${PROJECT_VERSION}${prerelease_string}")
-set(rpm-server-filename "foundationdb-server-${PROJECT_VERSION}${prerelease_string}")
+set(rpm-clients-filename "foundationdb-clients-${unversioned_postfix}")
+set(rpm-server-filename "foundationdb-server-${unversioned_postfix}")
+set(rpm-clients-versioned-filename "foundationdb${PROJECT_VERSION}-clients${prerelease_string}")
+set(rpm-server-versioned-filename "foundationdb${PROJECT_VERSION}-server${prerelease_string}")
```

Comment:
If I understand correctly, the names are going to be:

foundationdb-clients-6.3.20-1
foundationdb6.3.20-clients-1

If so, is it going to be confusing that the only difference is the position of the version? Perhaps we could add to the name a bit to disambiguate if we were concerned about this. E.g. foundationdb-versioned6.3.20-clients-1.

Reply:
this is only the base of the filename.  For 6.3.20 the filenames will be:
* `foundationdb6.3.20-clients-0.2dce835e960b0d9e3da4c28a3f5e9c909b83746d.PRERELEASE.versioned.x86_64.rpm`
* `foundationdb-server-6.3.20-0.2dce835e960b0d9e3da4c28a3f5e9c909b83746d.PRERELEASE.el7.x86_64.rpm`

So there is a `versioned`-string in there... I recognize this naming code is super ugly, but I would prefer to only fix this in master at another point in time.

Reply:
Ok, that sounds fine to me.